types/applicationinsights-js/index.d.ts

```diff
@@ -735,7 +735,8 @@ declare module Microsoft.ApplicationInsights {
          * @param   properties    map[string, string] - additional data used to filter events and metrics in the portal. Defaults to empty.
          * @param   measurements  map[string, number] - metrics associated with this event, displayed in Metrics Explorer on the portal. Defaults to empty.
          */
-        trackDependency(id: string, method: string, absoluteUrl: string, pathName: string, totalTime: number, success: boolean, resultCode: number, properties?: Object, measurements?: Object): any;
+        trackDependency(id: string, method: string, absoluteUrl: string, pathName: string, totalTime: number, success: boolean, resultCode: number,
```

Comment:
how does adding space help?

Reply:
tests failed otherwise because of linter requirements